Webex Connect Flows/v3.3/Template/Apple Messages for Business/README.MD

```diff
@@ -0,0 +1,77 @@
+# Overview of Apple Messages for Business template flows
+
+- This document covers the purpose and configuration details for the below template flows for Apple channel
+  - Inbound flow with form
+  - Unsubscribe flow
+
+## Overview of Apple Messages for Business inbound flow with form
+
+- This workflow will be triggered for every inbound customer message from the Apple Messages app.
+- The customer will be presented with a form asking for some basic details.
+- The fields in the form can be reconfigured by the flow admin as per their requirements.
+- The fields obtained from the form are extracted by using the Evaluate node and then made available to the agent by setting global/flow variables, using the Set Variable node.
+- Forms are supported only on iPhones and iPads by Apple, Mac devices do not have support.
+  - https://register.apple.com/resources/messages/msp-rest-api/type-interactive#form-message
+- Note: The chat widget on the agent's desktop will show the customer's name as was available the first time a message was sent from that customer to the business's asset.
```

Comment:
>The chat widget on the agent's desktop will show the customer's name as was

I think there is some typo here, please check

Reply:
There was no typo, however I have refactored it a little bit to be more clear